### PR TITLE
New small loader for new dashboard

### LINKED ISSUE #2988: New dashboard loading bar is not visible

### PR DESCRIPTION
**Pull request by xavijam**
### New small loader
It replaces the solid bar over the subheader, that was difficult to see:

loading

#### Bonus track:
- Fixed problem escapping avatar url in visualizations combo.

Fixes #2988.

### CODE REVIEW

**Comment by saleiva** [comment]
#JATORRISMO

**Comment by saleiva** [comment]
BTW, the separator line should be lighter. I fixed it and merged to master...

**Comment by piensaenpixel** [comment]
hahahahahaahaha

**Comment by xavijam** [comment]
@piensaenpixel forced me

**Comment by saleiva** [comment]
For me, it's too subtle, but I'm fine with giving this a try. A perfect example of aesthetics winning over function - :(

**Comment by piensaenpixel** [comment]
hahahaa OMG!

**Comment by xavijam** [comment]
Good thing is you can now change easy and painless.

**Comment by saleiva** [comment]
seriously, I think is too subtle. Merge and release anyway so we will try

**Comment by xavijam** [comment]
@viddo could you review this thing please?

File: app/assets/stylesheets/new_common/logo.css.scss [hunk 1]
```diff
@@ -41,3 +60,105 @@ $size: 40px;
...
+//   -o-animation-fill-mode: forwards;
+//   animation-fill-mode: forwards;
+//   @include animation(spin 0.7s linear 2);
+// }
+
+// @include keyframes(stop){
+//   to {
+//     @include transform(rotate(0deg));
+//   }
+// }
```
Comment (viddo) [inline app/assets/stylesheets/new_common/logo.css.scss]: These commented out lines can be removed, right?

Comment (viddo) [inline app/assets/stylesheets/new_common/logo.css.scss]: +the logo template below?

**Comment by viddo** [comment]
@xavijam apart from commented out stuff looks OK

**Comment by xavijam** [comment]
Remove commented css, yep, I don't know why it was still there. Thanks!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2848/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2849/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2853/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2859/))

**Pull request merged**